src/Agent.Worker/Build/TrackingManager.cs

```diff
@@ -506,7 +526,15 @@ private string GetGarbageFileLocation()
 
             // Update the info properties and save the file.
             config.UpdateJobRunProperties(executionContext);
-            WriteToFile(GetTrackingFileLocation(executionContext), config);
+
+            // Make sure we clean up any files in the old location (no job in the path)
```

Comment:
Comment is out of date ( "no job in the path" ).

Reply:
thanks for catching that!